modules/cnf-about-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,129 @@
+<3> Defines the specifics of the cluster updates.
+<4> The remediation plan starts with the clusters listed in the `canaries` field. Each canary cluster forms a single-cluster batch.
+<5> Defines the maximum number of concurrent updates in a batch. The number of remediation batches is the number of canary clusters, plus the number of clusters (except the canary clusters) divided by `maxConcurrency`. The clusters that are already compliant with all the managed policies are excluded from the remediation plan.
+<6> Displays information about the status of the updates.
+
+`UpgradeNotCompleted`:: In this state, the Operator enforces the policies following the remediation plan defined in the `UpgradeNotStarted` state.
++
+Enforcing the policies of subsequent batches starts immediately after all the clusters of the current batch are compliant with all the managed policies. If the batch times out, the Operator moves on to the next batch. The timeout value of a batch  is the `spec.timeout` field divided by the number of batches in the remediation plan.
++
+[NOTE]
+====
+The managed policies apply in the order they are listed in the `managedPolicies` field in the `ClusterGroupUpgrade` CR. One managed policy applies to the specified clusters at a time. After the clusters become compliant to the current policy, the next managed policy on the list will apply.
+====
++
+The Operator transitions to the `UpgradeTimedOut` state in two cases:
++
+* If the current batch is a canary type and the cluster in the batch does not change to compliant with all the managed policies.
+* If the clusters are not compliant with the managed policies within the `timeout` value specified in `remediationStrategy`.
+
+`UpgradeTimedOut`:: In this state, the Operator periodically checks if all the policies for the `ClusterGroupUpgrade` CR are compliant.
+The periodic checks allow the updates to complete if the updates are prolonged due to network, CPU, or other issues. If the policies are compliant, the Operator transitions to the `UpgradeCompleted` state.
+
+`UpgradeCompleted`:: In this state, the cluster updates are complete.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-upgrade-starts-complete
+  namespace: default
+spec:
+  deleteObjects: true <1>
+  clusters:
+  - spoke1
+  enable: false
+  managedPolicies:
+  - policy1-common-cluster-version-policy
+  - policy2-common-pao-sub-policy
+  remediationStrategy:
+    maxConcurrency: 1
+    timeout: 240
+status:
+  conditions:
+  - message: The ClusterGroupUpgrade CR has all clusters already compliant with the specified managed policies
+    reason: UpgradeCompleted
+    status: "False"
+    type: Ready
+----
+<1> The value of `deleteObjects` field is `true` by default. After the update, the Operator deletes the underlying {rh-rhacm} objects that were created during the update. The {rh-rhacm} Hub does not check for compliance after a successful update.
```

Comment:
```suggestion
<1> The value of `spec.action.afterCompletion.deleteObjects` field is `true` by default. After the update, the Operator deletes the underlying {rh-rhacm} objects that were created during the update. The {rh-rhacm} Hub does not check for compliance after a successful update.
```

Reply:
@irinamihai I'm checking, but I'm not sure where I found that statement. I see that in the repo it states:
> This is to avoid having RHACM Hub to continuously check for compliance since the upgrade has been successful.

Reworded the questionable sentence to:
> This option is to prevent the {rh-rhacm} Hub from continuously checking for compliance after a successful update.